### DISCUSSION TITLE
As an instance admin, is it possible to add people to organizations without being in the organization?

### DISCUSSION THREAD

**Discussion by eiclu** [dani-garcia/vaultwarden#3346] [Q&A]
Hi, we have a organization whose owners are no longer a part of the team. Now we would like to move the ownership of this organization to somebody still active. Is this possible somehow?

**Comment by stefan0xC** [comment]
Is somebody else a member of the organization? You can change their role in the [`/admin` panel](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page), when you click on the respective organization in the Users overview.
![vaultwarden-admin-panel-users](https://user-images.githubusercontent.com/509385/225281847-2bbd2a5c-db73-4419-a0f3-ffc1a17fa215.png)

Adding people to an organization itself is not possible.
